nucliadb_models/src/nucliadb_models/search.py

```diff
@@ -1351,9 +1351,15 @@ class AskRequest(AuditMetadataBase):
         ],
     )
     rag_images_strategies: list[RagImagesStrategies] = Field(
-        default=[],
+        default=[ParagraphImageStrategy()],
```

Comment:
Do we want to always have this strategy as default? All `/ask` requests will use it right?

Reply:
this was already the case before I started the changes (see the description)
just that the implementation was too implicit and, since there were no tests, I did break it